lib/moon/design/carousel/item.ex

```diff
@@ -22,9 +22,13 @@ defmodule Moon.Design.Carousel.Item do
   @doc "Will be got from Carousel in most cases"
   prop(is_active, :boolean, from_context: :is_active)
 
+  @doc "Conditional displaying, bc of some bug with named slots in surface"
```

Comment:
Maybe to change wording, eg:  "Conditional displaying due to an issue with named slots on Surface."